admin/access-control/password-reset.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Password Reset
+description: Learn how to reset Coder user passwords.
+---
+
+To reset a user's password:
+
+1. Go to **Manage** > **Users**.
+2. Find the user whose password you want to reset and click the vertical
+   ellipses to the right.
+3. Click **Reset password**. Coder will display a temporary password that you
+   can provide to the user.
```

Comment:
There's technically a confirmation dialog now (didn't used to be). They're shown the temporary password with cancel/reset and only if they click reset on the dialog is the action dispatched

![Screenshot from 2021-02-22 09-51-05](https://user-images.githubusercontent.com/11184711/108732778-7ee8e800-74f3-11eb-84ed-97517728b66f.png)